x/tokenomics/keeper/token_logic_modules.go

```diff
@@ -482,6 +495,93 @@ func (k Keeper) TokenLogicModuleGlobalMint(
 	return nil
 }
 
+// TokenLogicModuleGlobalMintReimbursementRequest processes the business logic
+// for the GlobalMintReimbursementRequest TLM.
+func (k Keeper) TokenLogicModuleGlobalMintReimbursementRequest(
+	ctx context.Context,
+	service *sharedtypes.Service,
+	sessionHeader *sessiontypes.SessionHeader,
+	application *apptypes.Application,
+	supplier *sharedtypes.Supplier,
+	actualSettlementCoin cosmostypes.Coin,
+	relayMiningDifficulty *servicetypes.RelayMiningDifficulty,
+) error {
+	logger := k.Logger().With("method", "TokenLogicModuleGlobalMintReimbursementRequest")
+
+	// Do not process the reimbursement request if there is no global inflation.
+	if MintPerClaimedTokenGlobalInflation == 0 {
+		logger.Warn("global inflation is set to zero. Skipping Global Mint Reimbursement Request TLM.")
+		return nil
+	}
+
+	// Determine how much new uPOKT to mint based on global inflation
+	newMintCoin, _ := calculateGlobalPerClaimMintInflationFromSettlementAmount(actualSettlementCoin)
+	if newMintCoin.Amount.Int64() == 0 {
+		return tokenomicstypes.ErrTokenomicsMintAmountZero
+	}
+
+	// Update the application's on-chain stake
+	// This should not fall below zero since `ensureClaimAmountLimits` should have
+	// already checked and adjusted the settlement amount so that the application
+	// stake covers the global inflation.
+	newAppStake := application.Stake.Sub(newMintCoin)
+	application.Stake = &newAppStake
+	logger.Info(fmt.Sprintf("updated application %q stake to %s", application.Address, newAppStake))
+
+	// Send the global per claim mint inflation uPOKT from the application module
+	// account to the tokenomics module account.
+	if err := k.bankKeeper.SendCoinsFromModuleToModule(
+		ctx, apptypes.ModuleName, tokenomicstypes.ModuleName, sdk.NewCoins(newMintCoin),
+	); err != nil {
+		return tokenomicstypes.ErrTokenomicsApplicationReimbursementRequestFailed.Wrapf(
+			"sending %s from the application module account to the tokenomics module account: %v",
+			newMintCoin, err,
+		)
+	}
+	logger.Info(fmt.Sprintf(
+		"sent (%s) from the application module account to the tokenomics module account",
+		newMintCoin,
+	))
+
+	// Send the global per claim mint inflation uPOKT from the tokenomics module
+	// account to PNF/DAO.
+	daoAccountAddr, err := cosmostypes.AccAddressFromBech32(k.GetAuthority())
```

Comment:
TL;DR, (It doesn't have to be in this PR but) we either need to align on an on-chain DAO address (see #889 for my proposal) OR align the dao_genesis_authorizations.json on the requirements of using the gov module account instead.

I saw something like this line somewhere else but have since lost track of it. This confuses me a bit because we've **opted to use authz to delegate authority over the (default - gov module, and configurable) "authority" account, instead of setting any particular address in the module config**. As such, it seems incorrect to me that we're sending the DAO's rewards to the governance module account.

This seems to me, at best, to be a temporary measure to be iterated on. Furthermore, we are currently missing an authz authorization in the dao_genesis_authorizations.json for the dao address to be able to do anything other than update parameters and manage upgrades on behalf of the gov module account. It stands to reason that if we're going to be holding DAO rewards in the gov module account, then we should probably add genesis authorizations for the DAO to use the `cosmos.bank.v1beta1.MsgSend` msg as well.